examples/text_labels_and_annotations/plot_tex.py

```diff
@@ -0,0 +1,39 @@
+"""
+=================================
+Rendering math equation using TeX
+=================================
+
+You can use TeX to render all of your matplotlib text if the rc
+parameter text.usetex is set.  This works currently on the agg and ps
+backends, and requires that you have tex and the other dependencies
+described at http://matplotlib.org/users/usetex.html
+properly installed on your system.  The first time you run a script
+you will see a lot of output from tex and associated tools.  The next
+time, the run may be silent, as a lot of the information is cached in
+~/.tex.cache
```

Comment:
I don't think we should mention the directory; it's an internal thing that can change, and I don't think this is even the right place on current systems.